tests/AwaitTest.php

```diff
@@ -10,6 +10,30 @@
 
 class AwaitTest extends TestCase
 {
+    /**
+     * @dataProvider provideAwaiters
+     */
+    public function testExecutionOrder(callable $await)
+    {
+        self::expectOutputString('acbd');
+
+        $deferred = new React\Promise\Deferred();
```

Comment:
We use `use operators` in our tests, this makes them easier to read and understand (especially longer tests)
```suggestion
        $deferred = new Deferred();
```
This occurs a few times in your test.